behavior.h

```diff
@@ -44,13 +44,13 @@ void behavior_start(int b);
 void behavior_stop(int b);
 int behavior_enabled(int b);
 
-#define B_ALL 			0xFFFF
-#define B_SOUND_BUTTON 		(1 << 0)
-#define B_LEDS_BUTTON 		(1 << 1)
+#define B_ALL			0xFFFF
+#define B_SOUND_BUTTON	(1 << 0)
```

Comment:
Indentation change, I do not know which one is correct. Probably because github indents by 8 and you indent by 4. Anyway the identation size should be defined and we should stick to it, configuring the editor to honour it. @retornaz, what is your convention for the firmawer?